src/styles/card.css

```diff
@@ -0,0 +1,103 @@
+.card {
+  max-width: 200px;
+  display: flex;
+  flex-direction: column;
+  background: var(--color-white);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+  
+.card__product-picture {
+  display: flex;
+  margin: 32px auto 40px;
+  width: 160px;
+  height: 134px;
+}
+ 
+.card__product-name {
+  margin: 0;
+  padding: 0 16px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-darkblue);
+}
+  
+.card__product-code {
+  margin: 4px 0 16px;
+  padding: 0 16px;
```

Comment:
could we set padding for the whole card instead of set for each element?

Reply:
yeah, way better
fixing it now